Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/patientdiscovery/response/TrustMode.java

```diff
@@ -66,7 +66,13 @@ public TrustMode() {
     public PRPAIN201306UV02 processResponse(ResponseParams params) {
         LOG.debug("Begin TrustMode.processResponse()...");
         PRPAIN201306UV02 response = null;
-
+        int sonargithub = 1;
+        //TODO: remove this
+        if (sonargithub ==2){
+            String removeLine = "remove me";
```

Comment:
The plugin will monitor and tackle the entire PR at once.  We hope to reduce sonarqube bug as we do PR.